scripts/wait-for-solr.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
 #
 # A helper script to wait for solr
-set -x
-until $(wget -O - http://localhost:8983 | grep -q -i solr); do
+until $(wget -q -O - http://localhost:8983 | grep -q -i solr); do
```

Comment:
This is currently taking the output of `wget | grep -q` and trying to run it as a command, so can be replaced with simply:
```bash
until wget -q -O - http://localhost:8983 | grep -q -i solr; do
```